docs/pages/core-concepts.mdx

```diff
@@ -78,27 +108,24 @@ PostgreSQL and MySQL.
 Read more about the [Teleport Database
 Service](./enroll-resources/database-access/database-access.mdx).
 
+### Teleport Discovery Service
+
+The `discovery_service` automates the process of finding and enrolling cloud resources such as AWS EC2 instances and Azure VMs into a Teleport cluster. It continuously scans supported cloud environments and adds matching resources based on configured rules, reducing the need for manual node registration. Read more about the [Teleport Discovery Service](./enroll-resources/auto-discovery/auto-discovery.mdx).
+
 ### Teleport Desktop Service
 
 Proxies Remote Desktop Protocol traffic to Windows desktops.
 
 Read more about the [Teleport Desktop
 Service](./enroll-resources/desktop-access/introduction.mdx).
 
-### Teleport Kubernetes Service
-
-Proxies HTTP traffic to the Kubernetes API server.
-
-Read more about the [Teleport Kubernetes
-Service](./enroll-resources/kubernetes-access/introduction.mdx)
+### Teleport Jamf Service
 
-### Teleport SSH Service
+The `jamf_service` integrates with Jamf to enforce device trust policies for macOS endpoints. Before a user is granted access, Teleport checks the device's compliance status with Jamf. This service is ideal for organizations that need to verify macOS device posture as part of their access control strategy, ensuring only trusted devices are allowed to connect. [Read more about the Jamf Service](./identity-governance/device-trust/jamf-integration.mdx).
```

Comment:
Nit: this service doesn't enforce device trust policies, it enrolls trusted devices. The enforcement of device trust policies is done elsewhere (mostly in the auth service).